.gitignore

```diff
@@ -6,4 +6,5 @@ blob-local-storage
 
 *.log
 
-*.DS_Store
\ No newline at end of file
+*.DS_Store
+src/plugins/SysML/SysMLExporter/BlockDefinitionDiagramExporter.js
```

Comment:
The merge attempt was actually made by @finalelement from my account in Thursday's MIC class and he is the one working on the exporter.